code/implementation/docker/storage/src/main/resources/META-INF/cattle/docker-storage/defaults.properties

```diff
@@ -1,5 +1,6 @@
 docker.index.url=https://index.docker.io
+docker.index.server=index.docker.io
```

Comment:
Add to code/implementation/docker/storage/src/test/resources/config.properties for unit tests to work